### DISCUSSION TITLE
Welcome to git-filter-repo Discussions!

### DISCUSSION THREAD

**Discussion by newren** [newren/git-filter-repo#180] [General]
## 👋 Welcome!
  GitHub suggested using Discussions as a place to connect with other members of the community. So I'm turning it on for this repository as a trial.  Here GitHub says you can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others, and their ideas with an open mind.
Remember that this is a community we build together 💪.

**Comment by pranav-bhart** [comment]
Hi, I see you reference the html file that contains the docs, but I cant see the html file in the repo.

**Reply by newren** [reply]
If you clone the repo, you get the sources, including the sources from which the documentation can be built.  The release tarballs have the build results, including the generated documentation.  See the "If you don't have the necessary documentation files (because you are installing from a clone of filter-repo instead of from a tarball)" portion of INSTALL.md.
